ff_time.c

```diff
@@ -141,7 +140,7 @@
 
     static portINLINE unsigned long ulDaysPerYear( int iYear )
```

Comment:
Shall we replace all occurrences of `unsigned long` in this module with `uint32_t`?
MISRA would complain about the ambiguity of `unsigned long`.

Reply:
Yes I would suggest that, but out of scope of this particular PR to reduce those specific compiler warnings.

Also using `size_t` and `uintptr_t` for sizes and addresses is also best.